make docs is broken

Now I can't build docs. It would be nice if we could avoid treating warnings as errors locally.
```
$ make docs
/Library/Developer/CommandLineTools/usr/bin/make -C docs html
Running Sphinx v1.6.3
loading intersphinx inventory from https://docs.python.org/objects.inv...
intersphinx inventory has moved: https://docs.python.org/objects.inv -> https://docs.python.org/2/objects.inv
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 9 source files that are out of date
updating environment: 9 added, 0 changed, 0 removed
reading sources... [100%] primitives

Warning, treated as error:
/Users/fritzobermeyer/github/uber/pyro/pyro/distributions/transformed_distribution.py:docstring of pyro.distributions.transformed_distribution.TransformedDistribution:1:Inline interpreted text or phrase reference start-string without end-string.
make[1]: *** [html] Error 1
make: *** [docs] Error 2
```

--- a/docs/Makefile
+++ b/docs/Makefile
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    = -E -W
+SPHINXOPTS    ?= -E
 SPHINXBUILD   = python -msphinx
 SPHINXPROJ    = Pyro
 SOURCEDIR     = source

